policies/test_test.rego

```diff
@@ -13,21 +13,31 @@ test_needs_tests_with_results {
 }
 
 test_needs_tests_with_results_mixed {
-	deny == {{"msg": "Found tests without results"}} with data.test as [{}, {"result": "SUCCESS"}]
+	deny == {{"msg": "Found tests without results"}} with data.test as [{}, {"test1": {"result": "SUCCESS"}}]
 }
 
 test_success_data {
-	count(deny) == 0 with data.test as [{"result": "SUCCESS"}]
+	count(deny) == 0 with data.test.test1 as {"result": "SUCCESS"}
 }
 
 test_failure_data {
-	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test as [{"result": "FAILURE"}]
+	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test.test1 as {"result": "FAILURE"}
+		with data.config.policy as {"non_blocking_checks": []}
```

Comment:
Had to add this, would be nice to be able to default this to `[]`, but `default data.config.policy.non_blocking_checks = []` is not a valid syntax